### PR TITLE
KFLUXBUGS-24: bump prod overlay chains deployment to use performance arguments

### PR DESCRIPTION
**Pull request by gabemontero**
Included PRs:
- https://github.com/openshift-pipelines/pipeline-service/pull/969
- https://github.com/openshift-pipelines/pipeline-service/pull/981

we've vetted https://github.com/redhat-appstudio/infra-deployments/pull/3496 in stage including tests from @lcarva 

though this also has @ralphbean 's tekton-ci to konflux-ci change with the pac gitauth secret

I think this is OK @enarha @ralphbean but with going to prod we probably need a sign off from you guys on that

rh-pre-commit.version: 2.2.0
rh-pre-commit.check-secrets: ENABLED

### CODE REVIEW

File: components/pipeline-service/production/stone-prd-rh01/deploy.yaml [hunk 1]
```diff
@@ -1541,7 +1541,7 @@ spec:
               set -o errexit
               set -o nounset
               set -o pipefail
-              for namespace in $(kubectl get namespaces -o name | cut -d/ -f2 | grep -E "\\-tenant$|^tekton-ci$|^konflux-ci$"); do
+              for namespace in $(kubectl get namespaces -o name | cut -d/ -f2 | grep -E "\\-tenant$|^tekton-ci$"); do
```
Comment (gabemontero) [inline components/pipeline-service/production/stone-prd-rh01/deploy.yaml:1544]: the recent namespace change

**Review by lcarva — APPROVED** [review]

**Comment by enarha** [comment]
I do not see the OSP bump here. It should look like https://github.com/redhat-appstudio/infra-deployments/pull/3503/files

**Comment by gabemontero** [comment]
> I do not see the OSP bump here. It should look like https://github.com/redhat-appstudio/infra-deployments/pull/3503/files

ok we got on the same page in slack

decision made to bump prod to the same pipeline-service level as stage, pulling in the new nightly

**Review by enarha — COMMENTED** [review]
/lgtm

**Comment by gabemontero** [comment]
/hold

per discussion in slack, I've told @enarha to release the hold, get someone to force merge it if we have unrelated flakes again

if all goes as planned, @lcarva should have a tuned chains controller when he logs in on Wednesday

**Pull request merged**